wireguard-install.sh

```diff
@@ -62,6 +81,71 @@ AllowedIPs = $CLIENT_WG_IPV4/32,$CLIENT_WG_IPV6/128" >>"/etc/wireguard/$SERVER_W
 	echo "It is also available in $HOME/$SERVER_WG_NIC-client-$CLIENT_NAME.conf"
 }
 
+function uninstall() {
+
+## To uninstall wireguard and its setup ##
+
+
+# Check OS type
+checkOS
+
+# get WireGuard interface name
+
+source /etc/wireguard/params
+
+# stop wireguard interface
+ip link set down "$SERVER_WG_NIC"	
+
+# Remove wireguard tool and qrencode
+if [[ $OS == 'ubuntu' ]]; then
+    apt-get remove --purge -y wireguard qrencode
+	add-apt-repository -y -r ppa:wireguard/wireguard
+	apt-get autoremove -y
+elif [[ $OS == 'debian' ]]; then
+	apt-get remove --purge -y wireguard qrencode
+	apt-get autoremove -y
+elif [[ $OS == 'fedora' ]]; then
+    dnf remove -y wireguard-tools qrencode
+	if [[ $VERSION_ID -lt 32 ]]; then
+		dnf remove -y wireguard-dkms
+		dnf copr disable -y jdoss/wireguard
+	fi
+	dnf autoremove -y
+elif [[ $OS == 'centos' ]]; then
+	yum -y remove wireguard-dkms wireguard-tools qrencode
+	yum -y autoremove
+elif [[ $OS == 'arch' ]]; then
+	pacman -Rs --noconfirm wireguard-tools wireguard-arch qrencode
+fi
+
+# Delete /etc/wireguard
+rm -rf /etc/wireguard > /dev/null 2>&1
+
+if [[ -e /etc/sysctl.d/wg.conf ]]; then
+    # Delete wg.conf
+    rm -f /etc/sysctl.d/wg.conf
+fi
```

Comment:
`rm -f /etc/sysctl.d/wg.conf` is enough, no `if` case required :smiley:

Reply:
Assuming in case file is already deleted or renamed, To prevent it from throwing error.

Reply:
If you run `rm` with `-f` it doesn't throw an error if the file doesn't exists.

Reply:
It's the same with
```bash
# Delete /etc/wireguard
rm -rf /etc/wireguard > /dev/null 2>&1
```

Reply:
Agree. Updated.